kuma/wiki/models.py

```diff
@@ -844,6 +844,9 @@ def revise(self, user, data, section_id=None):
             review_tags = ''
 
         new_rev.summary = data.get('summary', '')
+        
+        #To add comment, when Technical/Editorial review completed
```

Comment:
`kuma/wiki/models.py:861:9: E265 block comment should start with '# '` (needs a space after the `#`)